tools/dependency_summary.py

```diff
@@ -521,7 +521,7 @@
     "Apache-2.0": ["license-apache", "licence-apache"],
     "MIT": ["license-mit", "licence-mit"],
     "CC0": ["license-cc0", "licence-cc0"],
-
+    "Zlib": ["license-zlib", "license-zlib"],
```

Comment:
This helps the licenses summary script find the relevant license files automatically for Zlib.